V3-Upgrade-Guide.md

```diff
@@ -164,3 +165,12 @@ const myEntityApi = new MyEntityApi()
 the navigation properties are not correctly initialized leading to potential errors.
 To avoid this unintended usage of the constructor the visibility was changed to `private`.
 If you used the constructor directly please change your code to use the service function e.g. `myEntityService()` in the example above.
+
+### Update Transpilation options in Odata client generator
+
+The old `generateJs` option has been removed and instead replaced by `transpile`.
+By default, the OData generator will only generate TypeScript code.
+To generate JavaScript code, enable transpilation using the `--transpile` option.
+
+A new option, `tsConfig`, can be used to either pass a custom tsConfig configuration file or use a default config from the SDK.
```

Comment:
[pp] The name of the file is `tsconfig.json`, maybe we should mention it explicitly.
```suggestion
A new option, `tsConfig`, can be used to either pass a custom `tsconfig.json` configuration file or use a default config from the SDK.
```